Resources/views/process/show.html.twig

```diff
@@ -78,7 +78,9 @@
                     <div class="col-6 col-lg-3">
                         <dl class="row">
                             <dd class="col-sm-12 text-muted mb-0 pb-0">{{ 'process.table.court_decision'|trans }}</dd>
-                            <dt class="col-sm-12">{{ process.courtDecision ? 'court_decision.true'|trans : 'court_decision.false'|trans }}</dt>
+                            {% if process.courtDecision is not empty %}
```

Comment:
In Twig, `false` is empty (cf. https://twig.symfony.com/doc/2.x/tests/empty.html). `is null` is the way to check this (cf. https://twig.symfony.com/doc/2.x/tests/null.html).